internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -429,7 +429,7 @@ func LoadConfig(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionFu
 
 			b, e := fetchConfig(fetchCtx, r)
 			if errors.Is(e, perrors.ErrNoConfigSource) {
-				logger.Println("starting maintenance service")
+				logger.Println("config not found or empty, starting maintenance service")
```

Comment:
@Ulexus is our wordsmith, but should this be better called `machine configuration not found, starting maintenance service` ? (I believe in general "empty" is equivalent to the not found in the clouds, and for metal it's really not found)

Reply:
Agree; "machine configuration not found; starting maintenance service"

The semicolon is grammatical, but since semicolons are common delimiters, I'm fine with a comma, if we must.